tests/Benchmarks/Helpers/Server.cs

```diff
@@ -35,14 +35,14 @@ public static void RunSingleSegmentParser(int numberOfRequests, int concurrentCo
                             break;
                         }
 
-                        var requestBytes = input.First.Span;
+                        var requestBuffer = input.First;
 
-                        if (requestBytes.Length != 492)
+                        if (requestBuffer.Length != 492)
```

Comment:
Yes. Both have the same `Buffer<T>_length` value.

---
In reply to: [110496917](https://github.com/dotnet/corefxlab/pull/1454#discussion_r110496917) [](ancestors = 110496917)